### PR TITLE
Add {{path}} parsing to 'start-' and 'endtag'

### PR DESCRIPTION
**Pull request by meeroslav**
Now we can target specific files to inject based on their file path.
Works in alignment with 'relative' param to detect relative path from destination file.

This was added for easier partials loading in big application, with avoiding the need to have complex config structures.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -621,13 +621,14 @@ When `true` all tags without corresponding files will be emptied.
...
 Used to dynamically set starting placeholder tag depending on file extensions.
-In the provided string, or the string returned from the given function, the string `{{ext}}` is replaced with the source file extension name, e.g. "css", "js" or "html". `{{name}}` will be replaced by [`options.name`](#optionsname).
+In the provided string, or the string returned from the given function, the string `{{ext}}` is replaced with the source file extension name, e.g. "css", "js" or "html". `{{name}}` will be replaced by [`options.name`](#optionsname). `{{path}}` will be replaced by path to source file (when used together with `relative: true` it's relative path to file.
 
 ##### Default:
 
 A function dependent on target file type and source file type that returns:
 
 * html as target: `<!-- {{name}}:{{ext}} -->`
+* html as path: `<!-- {{path}} -->`
```
Comment (joakimbeng) [inline README.md]: That shouldn't be here. This lists the default tags for different target file types.
A complete usage example in the example section would be great!

File: README.md [hunk 2]
```diff
@@ -621,13 +621,14 @@ When `true` all tags without corresponding files will be emptied.
 **Purpose:**
 
 Used to dynamically set starting placeholder tag depending on file extensions.
-In the provided string, or the string returned from the given function, the string `{{ext}}` is replaced with the source file extension name, e.g. "css", "js" or "html". `{{name}}` will be replaced by [`options.name`](#optionsname).
+In the provided string, or the string returned from the given function, the string `{{ext}}` is replaced with the source file extension name, e.g. "css", "js" or "html". `{{name}}` will be replaced by [`options.name`](#optionsname). `{{path}}` will be replaced by path to source file (when used together with `relative: true` it's relative path to file.
```
Comment (joakimbeng) [inline README.md]: Please link to the `options.relative` section in the same way as for `options.name`

**Comment by joakimbeng** [comment]
Great, thanks!
See my comments for some small changes, otherwise it's LGTM!

**Comment by meeroslav** [comment]
Thanks for your quick reply. Please check my latest commit to verify readme file.

**Comment by okeegan** [comment]
@joakimbeng any chance we could get this merged in?

**Comment by joakimbeng** [comment]
Great job! Thanks :)

Also please don't hesitate in asking for being a collaborator, I need help maintaining this repo :)

**Comment by meeroslav** [comment]
I'll try to help whenever I have time.

**Pull request merged**